src/nft/components/explore/Explore.css.ts

```diff
@@ -0,0 +1,120 @@
+import { style } from '@vanilla-extract/css'
+import { breakpoints, sprinkles } from 'nft/css/sprinkles.css'
+
+/* Override for BodyWrapper properties */
+export const fullWidth = style([
+  sprinkles({
+    left: '1/2',
+    right: '1/2',
+    width: 'full',
+    maxWidth: 'full',
+    position: 'absolute',
+  }),
+  {
+    top: '72px',
+    marginLeft: '-50vw',
+    marginRight: '-50vw',
+  },
+])
+
+export const bannerWrap = style([
+  sprinkles({
+    position: 'relative',
+    overflow: 'hidden',
+    height: '386',
+  }),
+  {
+    backgroundPosition: 'center',
+    backgroundSize: 'cover',
+  },
+])
+
+export const bannerOverlay = style([
+  {
+    opacity: '0.7',
+    height: '386px',
+  },
+  sprinkles({
+    position: 'absolute',
+    zIndex: '0',
+    width: 'full',
+    backgroundColor: 'grey900',
+    left: '0',
+    top: '0',
+  }),
+])
+
+export const bannerContent = sprinkles({
+  position: 'relative',
+  zIndex: '1',
```

Comment:
confirming this is necessary--if so, should prefer a theme spec'd layer

Reply:
iirc the zIndex was necessary before the navbar changes - no longer needed